core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java

```diff
@@ -81,6 +82,11 @@
     private final NacosAsyncRestTemplate asyncRestTemplate = HttpClientBeanHolder
             .getNacosAsyncRestTemplate(Loggers.CORE);
     
+    /**
+     * Broadcast this node element information task.
+     */
+    private final MemberInfoReportTask infoReportTask = new MemberInfoReportTask();
```

Comment:
Please do not modify if there is no change.

Reply:
Ok